full.yml

```diff
@@ -555,17 +555,18 @@
     docker_image: "{{ drone_docker_image }}"
     docker_command: |
       /usr/bin/docker run --rm --name "{{ service_name }}" \
-      -v /var/run/docker.sock:/var/run/docker.sock \
       -v "{{ drone_directory.data }}":/data \
       -p "{{ drone_internal_web_port }}:80" \
       -e "USER_UID=1000" -e "USER_GID=1000" \
       -e "DRONE_GITEA_SERVER=https://{{ gitea_domain }}" \
+      -e "DRONE_GITEA_CLIENT_ID={{ gitea_oauth_client_id }}" \
+      -e "DRONE_GITEA_CLIENT_SECRET={{ gitea_oauth_client_secret }}" \
+      -e "DRONE_RPC_SECRET={{ drone_rpc_scret }}" \
       -e "DRONE_GIT_ALWAYS_AUTH=false" \
       -e "DRONE_RUNNER_CAPACITY=2" \
       -e "DRONE_SERVER_HOST={{ drone_domain }}" \
       -e "DRONE_SERVER_PROTO=https" \
       -e "DRONE_TLS_AUTOCERT=false" \
-      --network "{{ gitea_network }}" \
```

Comment:
¿Why? I think we should leave this network and add it to drone runner also for an easier and more secure setup.

Reply:
Its useless. With an external endpoint using oauth2 is not necessary to be on the same network. It not easier or secure due its useless.